presentation/src/main/java/com/sopt/presentation/auth/login/OnboardingRoute.kt

```diff
@@ -0,0 +1,154 @@
+        )
+    )
+
+    val pagerState = rememberPagerState(pageCount = { pages.size })
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(NoostakTheme.colors.white),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top
+    ) {
+        HorizontalPager(
+            state = pagerState
+        ) { page ->
+            Column(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.Top
+            ) {
+                Image(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    imageVector = ImageVector.vectorResource(id = pages[page].imageRes),
+                    contentDescription = null,
+                    contentScale = ContentScale.FillWidth
+                )
+                Text(
+                    modifier = Modifier
+                        .padding(top = 30.dp),
+                    text = pages[page].subtitle,
+                    color = NoostakTheme.colors.blue700,
+                    style = NoostakTheme.typography.b1SemiBold,
+                    textAlign = TextAlign.Center
+                )
+                Text(
+                    modifier = Modifier
+                        .padding(
+                            top = 8.dp,
+                            bottom = 26.3.dp
+                        ),
+                    text = pages[page].title,
+                    color = NoostakTheme.colors.gray900,
+                    style = NoostakTheme.typography.h322SemiBold,
+                    textAlign = TextAlign.Center
+                )
+            }
+        }
+        Row(
+            Modifier
+                .weight(1f)
+        ) {
+            repeat(pagerState.pageCount) { iteration ->
+                val color =
+                    if (pagerState.currentPage == iteration) NoostakTheme.colors.blue600 else NoostakTheme.colors.dotColor
+                Box(
+                    modifier = Modifier
+                        .padding(horizontal = 8.dp)
+                        .clip(CircleShape)
+                        .background(color)
+                        .size(8.dp)
+                )
+            }
+        }
+        NoostakBottomButton(
+            modifier = Modifier.padding(
+                horizontal = dimensionResource(id = R.dimen.horizontal_padding),
+                vertical = dimensionResource(id = R.dimen.vertical_padding)
+            ),
+            text = stringResource(R.string.btn_group_create_next),
+            activateColor = NoostakTheme.colors.blue600,
+            deactivateColor = NoostakTheme.colors.gray500,
+            onButtonClick = onNextButtonClick
+        )
+    }
+}
+
+data class BoardingPage(
+    val imageRes: Int,
+    val subtitle: String,
+    val title: String
+)
```

Comment:
P1: 불변하지 않는 애들이라 이 어노테이션도 추가해야될 것 같아요
```suggestion
@Immutable
data class BoardingPage(
    val imageRes: Int,
    val subtitle: String,
    val title: String
)
```